Solution1/BusinessLayer/Extensions/WeatherResponseDTOExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using BusinessLayer.DTOs;
+
+namespace BusinessLayer.Extensions
+{
+    public static class WeatherResponseDTOExtensions
+    {
+        public static string GetRepresentationSuccessResponse(this WeatherResponseDTO response)
+        {
+            return $"City: '{response.CityName}', Temp: {response.Temp}, Timer: {response.LeadTime} ms.";
```

Comment:
9, 14 strings almost same, can be put in private method

Reply:
+